tests/pipelines/pipeline/shared/pipelineconfig/test_transform.py

```diff
@@ -109,25 +59,33 @@ def test_get_transform_kwargs():
         "supplementary_distributions": [{"matches": "^data.xml$", "count": "1"}],
         "secondary_function": dataset_ingress_v1,
     }
-    result = get_transform_kwargs(config)
+    result = get_transform_details(config, "transform_kwargs")
     assert result == {"kwarg1": "value1"}
 
 
-def test_get_transform_kwargs_missing():
+def test_get_transform_details_missing():
+    """
+    Checks that get_transform_details returns the expected Assertion error when
+    the given input field does not exist in the config dictionary.
+    """
     config = {
         "config_version": 1,
         "transform": sdmx_compact_2_0_prototype_1,
-        "transform_inputs": {"^data.xml$": sdmx_sanity_check_v1},
+        "transform_kwargs": {},
         "required_files": [{"matches": "^data.xml$", "count": "1"}],
         "supplementary_distributions": [{"matches": "^data.xml$", "count": "1"}],
         "secondary_function": dataset_ingress_v1,
     }
     with pytest.raises(AssertionError) as err:
-        get_transform_kwargs(config)
-    assert "'transform_kwargs' not found in config dictionary" in str(err.value)
+        get_transform_details(config, "transform_inputs")
```

Comment:
Currently, the tests focus only on the missing transform_inputs. Should we consider expanding the test coverage to include cases where transform or transform_kwarg might be missing from the config dictionary?

Reply:
That can definitely be done if we want to be thorough with coverage. It would be doing the same thing as this test but checking a different string in the AssertionError.